Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -5624,8 +5624,8 @@ std::optional<LayoutUnit> RenderBox::explicitIntrinsicInnerWidth() const
         return std::nullopt;
 
     if (element() && style().containIntrinsicWidthType() == ContainIntrinsicSizeType::AutoAndLength && shouldSkipContent()) {
-        if (auto lastRememberedSize = element()->lastRememberedSize())
-            return isHorizontalWritingMode() ? LayoutUnit(lastRememberedSize->inlineSize()) : LayoutUnit(lastRememberedSize->blockSize());
+        if (auto width = isHorizontalWritingMode() ? element()->lastRememberedLogicalWidth() : element()->lastRememberedLogicalHeight(); width.has_value())
```

Comment:
I don't think you need `width.has_value()` since `std::optional` has `operator bool()`.

Reply:
Done